.gitattributes

```diff
@@ -7,4 +7,4 @@
 *.md    text
 *.txt   text
 *.tpl   text
-*.py    text diff=python
+*.py    text diff=python eol=lf
```

Comment:
Okay, explain this please.

The whole reason we added `.gitattributes` in the first place, was to provide windows users with CRLF conversion. What are we trying to do here? Why not just remove this file altogether?

I'm not trying to defend CRLF (it is an abhorrent abomination), but what are we trying to solve?

If Windows users generally have tools that can deal with LF line endings, why try to do CRLF conversion for some files?

And if they don't, why can we live with skipping this conversion for .py files?

I'm missing some consistency here guys...

Reply:
The issue we were looking to correct here is that if you are using Docker Desktop on Windows, the docker build runs on a linux based WSL container not your local machine. Git Desktop for Windows does auto-conversion to CRLF on all files, which causes `docker build .` to produce errors on launching the container. This was witnessed by @thomasjpatterson and @austinrdennis while trying to test the docker functions to build documentation. I am happy to roll this change out; however, we would need to potentially document how to address such errors if a user incurs them during the build of the docker image (should they choose to build their own rather than using the autobuild version).